src/public/modules/forms/admin/controllers/edit-fields-modal.client.controller.js

```diff
@@ -559,8 +560,8 @@ function EditFieldsModalController(
       field.uploadedFile = ''
       switch (ngfError.$error) {
         case 'maxSize':
-          vm.uploadError = `${(ngfError.size / 1000000).toFixed(2)} MB / ${
-            vm.maxImageSize / 1000000
+          vm.uploadError = `${(ngfError.size / MB).toFixed(2)} MB / ${
+            vm.maxImageSize / MB
```

Comment:
should we do a `toFixed` here as well so we don't show a large decimal to the user?

Reply:
While I can't be fully certain why the original author did not include `toFixed`, my best guess is that we expect `maxImageSize` to be divisible by `MB` (and it is). Please let me know if I have to add `toFixed` ultimately.

Reply:
nope no need, I'm okay to leave this as is